### PR TITLE
Update dependencies, prepare 2.2.7

### LINKED ISSUE #209: Update build_runner dependencies
Currently, this packages is only usable with build_runner ^1.8.0 (latest is build_runner 1.10.0).

Dependencies used need an update, For example build_runner_core dependency:
Used version: 4.1.0
Latest: 5.2.0

### PR DESCRIPTION
**Pull request by eernstg**
This PR is a small update that changes the dependencies (in particular, analyzer 0.40.3). All tests pass with all intermediate versions of the analyzer. The PR also adjusts the formatting (ran `dartfmt -w` on all), and removes some unused scripts from test_reflectable.

Fixes #209.

### CODE REVIEW

File: reflectable/example/example.dart [hunk 1]
```diff
@@ -22,7 +22,10 @@ class A {
   int argNamed(int x, int y, {int z = 42}) => x + y - z;
   int operator +(x) => 42 + x;
   int operator [](x) => 42 + x;
-  void operator []=(x, v) { f = x + v; }
+  void operator []=(x, v) {
+    f = x + v;
+  }
+
```
Comment (eernstg) [inline reflectable/example/example.dart:28]: Only `dartfmt` changes.

File: reflectable/example/example.dart [hunk 2]
```diff
@@ -50,7 +53,7 @@ main() {
   print(instanceMirror.invoke("arg1to3", [40, 2]));
   print(instanceMirror.invoke("arg1to3", [1, -1, 1]));
   print(instanceMirror.invoke("arg1to3", [21, 21, 0, "foo"]));
-  
+
```
Comment (eernstg) [inline reflectable/example/example.dart:56]: `dartfmt`

File: reflectable/lib/src/builder_implementation.dart [hunk 3]
```diff
@@ -5439,8 +5439,8 @@ Future<ResolvedLibraryResult> _getResolvedLibrary(
   // mentioned in dart-lang/build#2634. If we do not fetch a fresh session
   // then the code generation stops with an `InconsistentAnalysisException`
   // if there is more than one entry point.
-  final freshLibrary = await resolver
-      .libraryFor(await resolver.assetIdForElement(library));
+  final freshLibrary =
+      await resolver.libraryFor(await resolver.assetIdForElement(library));
```
Comment (eernstg) [inline reflectable/lib/src/builder_implementation.dart:5443]: `dartfmt`

File: test_reflectable/pubspec.yaml [hunk 4]
```diff
@@ -12,13 +12,9 @@ dependencies:
   build_runner: any
   glob: ^1.2.0
   reflectable:
-    git: git://github.com/dart-lang/reflectable.git
-    # git:
-    #   url: git://github.com/dart-lang/reflectable.git
-    #   ref: use_pedantic_jul19
-    # path: ../reflectable
+    path: ../reflectable
```
Comment (eernstg) [inline test_reflectable/pubspec.yaml:15]: Prepare for Travis test execution.

**Review by stefantsov — APPROVED** [review]

**Pull request merged**